Tasks/choco/Choco.ps1

```diff
@@ -62,28 +66,34 @@ function Install-Package
         [string] $IgnoreChecksums
     )
 
-    $expression = "$ChocoExePath install -y -f --acceptlicense --no-progress --stoponfirstfailure $Package --version $Version"
+    $expression = "$ChocoExePath install $Package"
+    
+    if ($Version){
+        $expression = "$expression --version $Version"
+    }
+
+    $expression = "$expression -y -f --acceptlicense --no-progress --stoponfirstfailure"
     
     if ($IgnoreChecksums -eq "true") {
         $expression = "$expression --ignorechecksums"
     }
 
-    Execute -Expression $expression
+    powershell.exe $expression
```

Comment:
Can you change this to a call to `Execute`?

Reply:
I tested a quick attempt of this yesterday, but it failed to work so I'll revisit this today.

Reply:
Try saving the expression to a temporary ps1 file, and provide that temporary file path to `Execute`.